### PR TITLE
translation of the home page into Portuguese

### PR DESCRIPTION
**Pull request by ElisaCapololo**
This PR is a  translation work

## Changes

- translation of the Wiki home page into Portuguese issue #83 

## Screenshots

home

## Checklist

- [ ] tested locally
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

Fixes #

### CODE REVIEW

File: Translations/Portuguese/Basic Information.md
Comment (antonio-pedro99) [inline Translations/Portuguese/Basic Information.md:1]: Please remove this file from this PR

File: Translations/Portuguese/Getting Started With Bolt For Javascript.md
Comment (antonio-pedro99) [inline Translations/Portuguese/Getting Started With Bolt For Javascript.md:1]: please remove this file from this PR

File: Translations/Portuguese/Home.md [hunk 1]
```diff
@@ -1,2 +1,9 @@
 This is the link  to the slack bot wiki homepage to be translated.
```
Comment (antonio-pedro99) [inline Translations/Portuguese/Home.md:1]: Please remove this line :)

**Review by antonio-pedro99 — CHANGES_REQUESTED** [review]
Hey @ElisaCapololo, nice work :)

Please note that when you're opening a new PR, you should create a new branch from the main branch. By doing so, you will avoid having unrelated changes in your PR. 

Please address my comments and we are ready to gooo.

File: Translations/Portuguese/Home.md (hunk 1 above)
Comment (ElisaCapololo) [reply Translations/Portuguese/Home.md]: I have already made the recommended changes

File: Translations/Portuguese/Getting Started With Bolt For Javascript.md
Comment (aosccode) [inline Translations/Portuguese/Getting Started With Bolt For Javascript.md:1]: Hi @ElisaCapololo, this file does not belong to here.

**Review by aosccode — CHANGES_REQUESTED** [review]
Hi hi @ElisaCapololo, I left some comments

File: Translations/Portuguese/Getting Started With Bolt For Javascript.md
Comment (ElisaCapololo) [reply Translations/Portuguese/Getting Started With Bolt For Javascript.md:1]: I have already removed the file

File: Translations/Portuguese/Home.md [hunk 2]
```diff
@@ -1,2 +1,8 @@
-This is the link  to the slack bot wiki homepage to be translated.
-[Home](https://github.com/chaoss/chaoss-slack-bot/wiki)
\ No newline at end of file
+[Home](https://github.com/chaoss/chaoss-slack-bot/wiki)
+
+
```
Comment (aosccode) [inline Translations/Portuguese/Home.md:3]: Please remove this extra line

**Review by aosccode — APPROVED** [review]
Please address this last comment and we are good :)

File: Translations/Portuguese/Home.md (hunk 2 above)
Comment (ElisaCapololo) [reply Translations/Portuguese/Home.md]: Done, thanks.

**Comment by Busayo-ojo** [comment]
Hi @ElisaCapololo, this PR has some conflicts, it seems you might have deleted some files, can you try to get this fixed?

**Comment by ElisaCapololo** [comment]
Yes, I can.

**Comment by ElisaCapololo** [comment]
Hello, I don't understand why the conflict still persists, can I close and open it again?

**Comment by misspee007** [comment]
Hey, I'm losing this because the changes are live [here](https://github.com/chaoss/chaoss-slack-bot/blob/main/Translations/Portuguese/Home.md) and on the [wiki](https://github.com/chaoss/chaoss-slack-bot/wiki/P%C3%A1gina-de-in%C3%ADcio).

**Pull request closed without merging**